torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -334,27 +334,27 @@ static auto registry = torch::RegisterOperators()
     // TODO Once callBoxed() supports optional tensor arguments, we can enable `use_c10_dispatcher: full` for backward()
     //      and remove the backend Autograd kernel here, only leaving the catch-all kernel.
     .kernel<decltype(VariableType::backward), &VariableType::backward>(DispatchKey::Autograd)
-    .catchAllKernel<decltype(VariableType::backward), &VariableType::backward>())
+    .kernel<decltype(VariableType::backward), &VariableType::backward>(DispatchKey::DefaultBackend))
```

Comment:
Why is this one and requires_grad_ below on a different Key?

Reply:
Ah because they have Autograd kernels, registering to Math with Autograd will override Autograd, so we register it to DefaultBackend to use them only in inference.

Reply:
Thanks for the reference! Interesting.